src/Orchard.Cms.Web/Modules/Orchard.OpenId/ViewModels/AuthorizeViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNetCore.Mvc.ModelBinding;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Orchard.OpenId.ViewModels
+{
+    public class AuthorizeViewModel
+    {
+        [Display(Name = "Application")]
```

Comment:
Don't use display properties and validation annotations if possible. Or at least don't rely on them to render text.